.travis.yml

```diff
@@ -25,6 +25,7 @@ python:
     - 3.5
     - 3.6
     - 3.7
+    - 3.8-dev
```

Comment:
Ah, sorry. It looks like numpy and other upstream packages aren't building manylinux wheels for 3.8, yet. That's fine.

```suggestion
```

Reply:
Ah thanks. I was just looking at the error and trying to figure out where I'd gone wrong. I'm not very familiar with travis to be honest. I've removed the line and repushed the commit.